docs/guides/runtime-options-overview.mdx

```diff
@@ -85,6 +86,7 @@ Scroll to see the full table.
 |                      |                          | shots_per_randomization |  integer >= 1                                                                         |  `128`                    |                      |
 |                      |                          | num_randomizations      |   integer >= 1                                                                            |  `32`                    |                      |
 |                      |                          | layer_pair_depths       |  list[int] of 2-10 values in the range [0, 200]                           | `(0, 1, 2, 4, 16, 32)`              |                      |
+|                      | [layer_noise_model](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2#layer_noise_model)       |        | `UnsetType`/`NoiseLearnerResult`/`Sequence[LayerError]`                      | `Unset`                      |                      |
```

Comment:
```suggestion
|                      | [layer_noise_model](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2#layer_noise_model)       |        | `NoiseLearnerResult`/`Sequence[LayerError]`                      | `None`                      |                      |
```
We don't really list `UnsetType`, since it's not a real choice. It's just a place holder to indicate you haven't made a selection. And the default should be None (the API doc is wrong).